packages/client/src/revenue/Revenue.ts

```diff
@@ -115,6 +121,7 @@ export class Revenue {
         {
           request,
           observerId,
+          ...buildMediaTransformsFromConfig(this.config.mediaTransforms),
```

Comment:
I wonder if there's a nice way to abstract this enriching of the request into a function `...buildMediaTransformsFromConfig(this.config.mediaTransforms)` and keep this logic contained into one place?

Reply:
`enrichRequestWithMediaTransformations(...)` maybe?

Reply:
discussed with @krzysu, difficult to abstract any further due to media transformations not always being required